grammar.js

```diff
@@ -1065,12 +1066,66 @@ module.exports = grammar({
       optional($.limit),
     ),
 
-    _table_expression: $ => seq(
-      $.table_expression,
+    relation_list: $ => seq(
+      choice(
+        $.relation,
+      ),
       repeat(
         seq(
           ',',
-          $.table_expression,
+          choice(
+            $.relation,
+          )
+        ),
+      ),
+    ),
+
+    _table_expression: $ => seq(
+      optional(
+        $.keyword_only,
+      ),
+      optional(
+        seq(
+          field('schema', $.identifier),
+          '.',
+        ),
+      ),
+      field('name', $.identifier),
+    ),
+
+    _inplace_insert: $ => seq(
```

Comment:
this could be reusable for `insert_expression` as a `values` node (also a better name, imo) if `relation` offered a choice `seq('(', $.values, ')'`